.travis.yml

```diff
@@ -31,6 +31,8 @@ matrix:
     addons: {apt: {packages: [libgmp-dev]}}
   - env: BUILD=stack STACK='stack --stack-yaml=stack-9.0.yaml'
     addons: {apt: {packages: [libgmp-dev]}}
+  - env: BUILD=stack STACK='stack --stack-yaml=stack-9.2.yaml'
```

Comment:
We switched to github actions for CI. Please update  `.github/workflows/stack-ci.yml`.

Could you do me favor and delete `.travis.yml`? It only confuses us...

Reply:
Seems like I didn't need to change anything in `stack-ci.yml`? Just `matrix.yaml`? (Not sure why matrix is defined in another file; why not just define as a normal matrix in the github actions config?)